overview/index.md

```diff
@@ -3,11 +3,11 @@ layout: project
 title: About the DevOps Dojo Consortium
 ---
 
-The DevOps Dojo Consortium is collaboration of enterprise Dojos all seeking to improve both the outcomes of their companies and the
+We are collaboration a of enterprise Dojos all seeking to improve both the outcomes of their companies and the
```

Comment:
placement of 'a' is wrong at beginning of sentence

Reply:
+1